tests/common/plans/test_fast_scan.py

```diff
@@ -89,10 +93,12 @@ def capture_emitted(name, doc):
     assert_emitted(docs, start=1, descriptor=1, event=mock.ANY, stop=1)
 
 
-async def test_fast_scan_2d_success(sim_motor: XYZStage, RE: RunEngine, det):
+async def test_fast_scan_2d_success(
+    sim_motor: XYZStage, RE: RunEngine, det: SynPeriodicSignal
+) -> None:
```

Comment:
Should: This is an other historical relic. SynPeriodicSignal is from ophyic, they have the read function so they are readable they just never declare it, we can do this so we do not need type ignore
```
    sim_motor: XYZStage, RE: RunEngine, det: Readable
) -> None:
```
or better yet we remove the ophyd dependence
 
```python 
from tests.sim_devices import SimDetector
```
```suggestion
    sim_motor: XYZStage, RE: RunEngine, fake_detector: SimDetector
) -> None:
```

Reply:
I would prefer `Readable` because the device is not a `SimDetector` and can be confusing reading it as such

Reply:
Actually, no. The reason I did this is because `det.start_simulation()`, so you have to declare it as type `SynPeriodicSignal`